recipe_scrapers/dobruchutaktualitysk.py

```diff
@@ -0,0 +1,52 @@
+from ._abstract import AbstractScraper
+
+
+class DobruChutAktualitySK(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "dobruchut.aktuality.sk"
+
+    def keywords(self):
+        # site has no support for recipe keywords
+        return list()
```

Comment:
```suggestion
    def keywords(self):
        # site has no support for recipe keywords
        return None
```

Reply:
Hm.  In fact: raising an `FieldNotProvidedByWebsiteException` here might be better.